lib/Models/WebMapServiceCatalogItem.js

```diff
@@ -377,12 +528,14 @@ WebMapServiceCatalogItem.prototype.updateFromCapabilities = function(capabilitie
 
         if (defined(this.layers)) {
             var layers = this.layers.split(',');
+            var styles = (this.styles || this.parameters.styles || '').split(',');
```

Comment:
Why do you define `styles` as a string with commas in it, rather than an array? Is it to match up with `layers`?